ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -80,8 +80,21 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
-- as: aws-ipi-disconnected-cco-manual-sts-p2-f14
-  cron: 17 21 8,23 * *
+- as: aws-ipi-disconnected-cco-manual-sts-ep-p2-f7
```

Comment:
can we keep the original test frequency - f14?

Reply:
ahh, you've decreased the frequency https://github.com/openshift/release/pull/36490,  I copied it from the previous version. Okay, will sync up with your changes.

Reply:
Yeah, even per our agreement, if the install workflow is covered in both arm and amd, we even can set it to f28 for 4.10/4.11. But maybe deserve a separate PR for that.